creator/styled_paragraph.go

```diff
@@ -226,6 +226,48 @@ func (p *StyledParagraph) Height() float64 {
 	return height
 }
 
+// getLineHeight returns both the capheight and the font size based height of
+// the line with the specified index.
+func (p *StyledParagraph) getLineHeight(idx int) (capHeight, height float64) {
+	if p.lines == nil || len(p.lines) == 0 {
+		p.wrapText()
+	}
+	if idx < 0 || idx > len(p.lines)-1 {
+		return 0, 0
```

Comment:
Output a debug message that it is out of range.

Reply:
Done